src/store/modules/session.js

```diff
@@ -45,6 +54,7 @@ const mutations = {
   setPing: set("ping"),
   setVotingSpeed: set("votingSpeed"),
   setVoteInProgress: set("isVoteInProgress"),
+  toggleRecordVoteHistory: toggle("recordVoteHistory"),
```

Comment:
```suggestion
  setVoteHistoryAllowed: set("isVoteHistoryAllowed"),
```
Given that you pass in a value in 2 of the 3 use cases anyhow, I'd just add the explicit value to the third use case (Menu.vue) as well to keep it more in line with the other boolean props here.